src/NodaTime/YearMonthDay.cs

```diff
@@ -12,10 +12,11 @@ namespace NodaTime
     /// </summary>
     /// <remarks>
     /// <para>
-    /// The day is represented in bits 0-5. The month is represented in bits 6-9. The
-    /// year is represented in bits 10-24. This type is naive: comparisons are performed
-    /// assuming that a larger month number always comes after a smaller month number,
-    /// etc. This is suitable for most, but not all, calendar systems.
+    /// See <see cref="YearMonthDayCalendar"/> for the number of bits per component,
+    /// but this type shifts all information over (so bit 0 is part of the day value).
```

Comment:
perhaps something like "but this type excludes the calendar field (so that bit 0..."